app/src/interfaces/input-rich-text-html/input-rich-text-html.vue

```diff
@@ -421,14 +422,21 @@ export default defineComponent({
 			count.value = iframeContents?.textContent?.replace('\n', '')?.length ?? 0;
 		}
 
+		function contentUpdated() {
+			setCount();
+
+			if (!observer) return;
+
+			emit('input', editorRef.value.getContent() ? editorRef.value.getContent() : null);
+		}
+
 		function setupContentWatcher() {
 			if (observer) return;
 
 			const iframeContents = editorRef.value.contentWindow.document.getElementById('tinymce');
 
 			observer = new MutationObserver((_mutations) => {
-				count.value = iframeContents?.textContent?.replace('\n', '')?.length ?? 0;
-				emit('input', editorRef.value.getContent() ? editorRef.value.getContent() : null);
+				contentUpdated();
```

Comment:
Hey @licitdev 
Someone reported that pasting code in WYSIWYG inside translations duplicates entries.
Can you give a look? 
I tested it and it seems it comes from here. I think adding a `debounce` will solve it but I wonder if we can filter the proper mutation using `_mutations` 🤔

Reply:
Thanks! Can confirm the issue when pasting content, there's indeed multiple firing of event. There's this TinyMCE [issue](https://github.com/tinymce/tinymce/issues/4471). Posted the issue at #14215.